{
    "id": "bundle--f3a59207-507c-4b38-8350-c48c957b5829",
    "objects": [
        {
            "created": "2021-03-13T20:12:20.264815Z",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "description": "An out-of-bounds read was addressed with improved input validation. This issue is fixed in macOS Catalina 10.15.3, watchOS 6.1.2. A remote attacker may be able to cause unexpected application termination or arbitrary code execution.",
            "external_references": [
                {
                    "external_id": "CVE-2020-3877",
                    "source_name": "cve"
                }
            ],
            "id": "vulnerability--732071d5-4e5d-4711-8472-b0fb2a2a8c73",
            "modified": "2021-11-22T14:54:52.900149Z",
            "name": "CVE-2020-3877",
            "spec_version": "2.1",
            "type": "vulnerability"
        }
    ],
    "type": "bundle"
}